src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/cas1/Cas1AssessmentDomainEventServiceTest.kt

```diff
@@ -154,6 +159,7 @@ class Cas1AssessmentDomainEventServiceTest {
           withArg {
             Assertions.assertThat(it.data.eventDetails.allocatedBy).isNull()
           },
+          any(),
```

Comment:
i think this should be checking that user is TriggerSourceType.SYSTEM, given the test name? It might be better to rename the test to 'assessmentAllocated allocating user is system'?